### PR TITLE
[core-3881] add documentation for certifications

### PR DESCRIPTION
**Pull request by radekzawada**
https://bookingsync.atlassian.net/browse/CORE-3881

### CODE REVIEW

File: content/reference/enums.md [hunk 1]
```diff
@@ -380,3 +380,13 @@ parity           | For parity information
...
 {: class="table table-bordered"}
+
+## Rental Certifications
+
+Key                 | Allowed values
+--------------------|------------
+meubles_de_tourisme | 1_etoile 2_etoiles 3_etoiles 4_etoiles 5_etoiles
+gites_de_france     | 1_epi 2_epis 3_epis 4_epis
+clevacances         | 1_cle 2_cles 3_cles 4_cles
+accueil_velo        | accueil_velo
```
Comment (thilonel) [inline content/reference/enums.md:391]: @radekzawada should we say that they also allow `""` values for these keys?

**Review by thilonel — APPROVED** [review]
Just one question.

File: content/reference/enums.md (hunk 1 above)
Comment (radekzawada) [reply content/reference/enums.md:391]: validation allows to have empty string, wondering how to describe that correctly here, do you have any idea ?

Comment (thilonel) [reply content/reference/enums.md:391]: nah :/ anyhow, let's just go with this. Could you please merge and deploy?

Comment (radekzawada) [reply content/reference/enums.md:391]: on it

**Pull request merged**